<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfilerWindow</class>
 <widget class="QMainWindow" name="ProfilerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ANNarchy Profiler</string>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="cmbThread">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="AnalyzerWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="deviation_chart">
       <attribute name="title">
        <string>Standardabweichung</string>
       </attribute>
       <layout class="QHBoxLayout" name="errorbar_chart_layout">
        <item>
         <widget class="ErrorbarChartTree" name="ErrorbarChartTree">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>100</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="ErrorbarChartWidget" name="ErrorbarChart" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="PieChartTab">
       <attribute name="title">
        <string>Torten</string>
       </attribute>
       <layout class="QHBoxLayout" name="pie_chart_layout">
        <item>
         <widget class="PieChartTree" name="PieChartTree">
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="headerStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="PieChartWidget" name="PieChart" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>787</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStart">
    <property name="title">
     <string>Start</string>
    </property>
    <addaction name="btnLoadData"/>
    <addaction name="btnRunMeasurement"/>
    <addaction name="btnSave"/>
   </widget>
   <addaction name="menuStart"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="btnLoadData">
   <property name="text">
    <string>Load data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="btnRunMeasurement">
   <property name="text">
    <string>Run measurement</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="btnSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PieChartWidget</class>
   <extends>QWidget</extends>
   <header>Charts</header>
   <container>1</container>
   <slots>
    <slot>draw(PyQt_PyObject)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>PieChartTree</class>
   <extends>QTreeWidget</extends>
   <header>Analyzer</header>
   <slots>
    <signal>drawPieChart(PyQt_PyObject,PyQt_PyObject,PyQt_PyObject)</signal>
    <slot>load_data(PyQt_PyObject)</slot>
    <slot>current_item_changed(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>ErrorbarChartTree</class>
   <extends>QTreeWidget</extends>
   <header>Analyzer</header>
   <slots>
    <signal>drawErrorbarChart(PyQt_PyObject,PyQt_PyObject)</signal>
    <slot>load_data(PyQt_PyObject)</slot>
    <slot>current_item_changed(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>ErrorbarChartWidget</class>
   <extends>QWidget</extends>
   <header>Charts</header>
   <container>1</container>
   <slots>
    <slot>draw(PyQt_PyObject,PyQt_PyObject)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>PieChartTree</sender>
   <signal>drawPieChart(PyQt_PyObject,PyQt_PyObject,PyQt_PyObject)</signal>
   <receiver>PieChart</receiver>
   <slot>draw(PyQt_PyObject)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>761</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ErrorbarChartTree</sender>
   <signal>currentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</signal>
   <receiver>ErrorbarChartTree</receiver>
   <slot>current_item_changed(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ErrorbarChartTree</sender>
   <signal>drawErrorbarChart(PyQt_PyObject,PyQt_PyObject)</signal>
   <receiver>ErrorbarChart</receiver>
   <slot>draw(PyQt_PyObject,PyQt_PyObject)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>761</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PieChartTree</sender>
   <signal>currentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</signal>
   <receiver>PieChartTree</receiver>
   <slot>current_item_changed(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
